README.md

```diff
@@ -787,6 +788,21 @@ You can read more about the inference request parameters in the [parameters
 extension](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_parameters.md)
 documentation.
 
+## Inference Response Parameters
+
+Inference response parameters may be optionally set during the construction of
+an inference response object. The parameters should be a dictionary of key value
+pairs, where keys are `str` and values are `bool`, `int` or `str`. For example,
+```python
+response = pb_utils.InferenceResponse(
+    output_tensors, parameters={"key": "value"}
+)
+```
+
+You can read more about the inference response parameters in the [parameters
+extension](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_parameters.md)
+documentation.
```

Comment:
Let's add a note here on the limitation with BLS models until implemented

Reply:
Sure, added [[Docs] Note BLS response parameters are not populated currently](https://github.com/triton-inference-server/python_backend/pull/394/commits/47e71e0c6334e4b8fa1f491db47a1782c739e910)